editor.planx.uk/tsconfig.json

```diff
@@ -16,8 +16,9 @@
     "skipLibCheck": true,
     "strict": true,
     "target": "es6",
-    "typeRoots": ["src/types"]
+    "typeRoots": ["src/types"],
+    "types": ["vite/client", "vite-plugin-svgr/client"]
   },
-  "include": ["**/*.ts", "**/*.tsx"],
+  "include": ["**/*.ts", "**/*.tsx", "./src/setupTests.ts"],
```

Comment:
nit: I explicitly added `"./src/setupTests.ts"` at some point here when troubleshooting, but I don't actually think we need it / it should already be picked up by `"**/*.ts"` ? Can you remove and double-check please?

Reply:
Working locally without this, just pushed up the change 👍